main.go

```diff
@@ -110,10 +116,12 @@ func main() {
 
 	// Set up a ClusterCacheTracker and ClusterCacheReconciler to provide to controllers
 	// requiring a connection to a remote cluster
+	var log logr.Logger
+
 	tracker, err := remote.NewClusterCacheTracker(
 		mgr,
 		remote.ClusterCacheTrackerOptions{
-			Log:     ctrl.Log.WithName("remote").WithName("ClusterCacheTracker"),
+			Log:     &log,
```

Comment:
let's create a separate log like before WithName("remote").WithName("ClusterCacheTracker")

if we don't want to use ctrl.log we can use 
log.WithName("remote").WithName("ClusterCacheTracker")

Reply:
Fixed